src/components/PaymentSuccessBox.jsx

```diff
@@ -0,0 +1,25 @@
+import { Box, Typography } from '@mui/material'
+import React from 'react'
+
+const PaymentSuccessBox = ({orderNO,address,orderItem}) => {
+  return (
+    <Box sx={{maxWidth:"70%",display:"flex",flexDirection:"column",gap:"15px"}}>
+       <Typography variant='h2'sx={{color:"#242424"}}>Payment Success !</Typography>
+       <div style={{display:"flex",justifyContent:"space-between"}}>
```

Comment:
This element is duplicated multiple times through the component, 
it'd be better if we created a component and passed the title and the value as props